src/core/load-fragment-gperf.gperf.m4

```diff
@@ -202,6 +202,8 @@ $1.DisableControllers,           config_parse_disable_controllers,   0,
 $1.IPAccounting,                 config_parse_bool,                  0,                             offsetof($1, cgroup_context.ip_accounting)
 $1.IPAddressAllow,               config_parse_ip_address_access,     0,                             offsetof($1, cgroup_context.ip_address_allow)
 $1.IPAddressDeny,                config_parse_ip_address_access,     0,                             offsetof($1, cgroup_context.ip_address_deny)
+$1.IPFilterIngress,              config_parse_ip_filter_bpf_progs,   0,                             offsetof($1, cgroup_context.ip_filters_ingress)
+$1.IPFilterEgress,               config_parse_ip_filter_bpf_progs,   0,                             offsetof($1, cgroup_context.ip_filters_egress)
```

Comment:
So, I'd put more focus on the fact we specify programs by fs path here. i.e. maybe name this IPIngressFilterPath= or so. This would then nicely match NetworkNamespacePath= which semantically is very similar I think. I.e. what differentiates this new option from IPAddressAllow= and IPAddressDeny= is not whether its a filter or not (because all of them are), but that we specify the filter by path in fs here.

Reply:
Yes, changed!